DigitalOffice/Services/CompanyService/Repositories/CompanyRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LT.DigitalOffice.CompanyService.Database;
+using LT.DigitalOffice.CompanyService.Database.Entities;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Linq;
+
+namespace LT.DigitalOffice.CompanyService.Repositories
+{
+    public class CompanyRepository : ICompanyRepository
+    {
+        private readonly CompanyServiceDbContext dbContext;
+
+        public CompanyRepository([FromServices] CompanyServiceDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+
+        public DbPosition GetPositionById(Guid positionId)
+        {
+            var position = dbContext.Positions.FirstOrDefault(position => position.Id == positionId);
```

Comment:
Rename position variable(to dbPosition, for example), because parameter 'position' hides outer local variable with the same name

Reply:
Fixed